lib/site_template/README.txt

```diff
@@ -0,0 +1,23 @@
+===============================
+Welcome to your new Jekyll Site
+===============================
+
+This file generated by Jekyll will neither be included in the site built
+when you run `jekyll build` (or `jekyll serve`) nor will it be read by
+git if you plan to have your site set up as a git repository.
+
+For the new users of Jekyll, Welcome to Jekyll! :) The directory you just
+generated is a self-sufficient blog. You should be able to view this blog
+in your browser by simply running `bundle exec jekyll serve` in your
+command-terminal and pointing your browser to http://localhost:4000/. For
+more information on using Jekyll, goto: https://jekyllrb.com/docs/home/
+
+For the experienced users of Jekyll who just recently upgraded from v3.1.x
+and older, don't be confused by the missing folders in your blog. They've
+been bundled together as a theme-gem called 'minima' which your site will
+look-up to during the 'jekyll build' process. This configuration can be
+changed by editing the "theme: minima" line in your '_config.yml' and the
```

Comment:
Let's be consistent with either single or double quotes, not both unless they're nested in which case the double quotes are on the outside and single quotes inside.